node/rpc/Cargo.toml

```diff
@@ -60,6 +56,8 @@ state_tree = { path = "../../vm/state_tree" }
 wallet = { package = "key_management", path = "../../key_management", features = [
     "json",
 ] }
+forest_crypto = { version = "0.5.2", features = ["blst", "json"] }
+fil_actors_runtime = "=7.5.1"
```

Comment:
Like before, this should be deleted or have a different version number.

Reply:
Should be good now